caramel/module/scripts/caramel.handlebars.js

```diff
@@ -606,7 +606,9 @@ engine('handlebars', (function () {
                                     o = helper.resources(page, meta);
                                     blockData.resources.js = resolve('js', o.js);
                                     blockData.resources.css = resolve('css', o.css);
-                                    blockData.resources.code = o.code ? evalCode(o.code, meta.data, theme) : null;
+                                    blockData.resources.code = o.code ? o.code.reduce(function(pre,cur){
```

Comment:
Thanks @mushthaq33 , I have fixed it in [this commit](https://github.com/wso2/jaggery-extensions/pull/83/commits/aadb1f42e52e47641b2c9b82dd4154bc0ef47534)